Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -93,6 +93,7 @@ class TopTabsViewController: UIViewController {
         super.viewWillAppear(animated)
         collectionView.dataSource = self
         collectionView.delegate = tabLayoutDelegate
+        collectionView.reloadData()
```

Comment:
Since the collectionView.reloadData() is also called in viewDidAppear, it might be better to delete second reloadData() in viewDidAppear to reduce the extra reloading.